db/schema.rb

```diff
@@ -147,6 +147,7 @@
     t.string   "first_name",                             null: false
     t.string   "last_name",                              null: false
     t.boolean  "is_admin",               default: false, null: false
+    t.boolean  "has_sidebar",            default: false, null: false
```

Comment:
Could we default this to `true`, since at the very beginning of a user's existence we should probably show the sidebar.